### PR TITLE
Add combo network to Chianlist

### PR DESCRIPTION
**Pull request by combo-labs**
add combo mainnet and testnet

### CODE REVIEW

**Comment by combo-labs** [comment]
@ligi  Please help to review it, we urgently need promotion.

**Review by RAPRIT9134 — COMMENTED** [review]
combo-labs:master
[0xE1fE80656797f031F40dD1eCb3ec298F5E79b2fe](url)

File: _data/chains/eip155-91715.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
[the remark this answers is not in the document]
Comment (Amirmortazavi61) [reply _data/chains/eip155-91715.json:1]: Tanks

File: _data/chains/eip155-91715.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "Combo Testnet",
```
[the remark this answers is not in the document]
Comment (Amirmortazavi61) [reply _data/chains/eip155-91715.json:2]: Tanks

**Comment by combo-labs** [comment]
update

**Comment by combo-labs** [comment]
@ligi  Please help merge or let me know where modifications are needed.

**Comment by combo-labs** [comment]
@ligi  Thank you for your review. Thank you for your review. I have modified the position of the icon in the file.

**Comment by combo-labs** [comment]
@ligi Thank you for your review. I delete combo testnet.

**Comment by combo-labs** [comment]
2151702721713_ pic

@ligi  Thank you for your review.

**Comment by combo-labs** [comment]
@ligi  Thank you for your review.

**Comment by SYLing** [comment]
@ligi Thank you for your review. We have made the changes, please help merge them.

**Comment by combo-labs** [comment]
@ligi  Thank you for your review. We have made the changes, please help merge them.

**Comment by combo-labs** [comment]
@ligi  I have made the changes, please help me check them.

**Comment by defusionai** [comment]
@ligi I have made the changes, please help me check them.

**Comment by combo-labs** [comment]
@ligi   I have made the changes.

**Review by ligi — APPROVED** [review]

**Pull request merged**